docker/test/integration/MiNiFi_integration_test_driver.py

```diff
@@ -266,3 +266,10 @@ def check_azure_storage_server_data(self, cluster_name, object_data):
     def wait_for_kafka_consumer_to_be_registered(self, cluster_name):
         cluster = self.acquire_cluster(cluster_name)
         assert cluster.wait_for_kafka_consumer_to_be_registered()
+
+    def check_minifi_log_contents(self, line):
+        line_found = False
+        for cluster in self.clusters.values():
+            if cluster.get_engine() == "minifi-cpp":
+                line_found = cluster.wait_for_app_logs(line, 60)
+        assert line_found
```

Comment:
Currently it would be better to have the cluster_name, although there may be still multiple minifi containers in a single cluster. After #1121 is merged it will be much simpler as there will only be a single cluster and you can specify the container you want to get the logs from.

Reply:
I think it's OK to leave it like this for now, as it will need to be modified when resolving the conflict with #1121, whichever PR goes in first.